boards/nxp/mcx_nx4x_evk/board.c

```diff
@@ -18,14 +18,16 @@
 #define BOARD_USB_PHY_TXCAL45DM (0x07U)
 
```

Comment:
A commit message should be focused on *why* the changes is needed and less on *what* is does. What it does should be somehow obvious from the code (if it is not then explain it briefly in the commit message).

So here your commit message should be something like this:

```
 board: mcx_nx4x: Fix formatting 

In order to make style more consistent and code easier to read
fix some style issues:
- add enum values each on a single line
- fixes multiple whitespaces in macro definitions

Signed-off-by: Tomas Barak <tomas.barak@nxp.com>
```

If those were detected by a tool (e.g checkpatch.pl) please also mention this in the commit message.


Also, this change can be marked as trivial sent in a different PR and merged faster your if current PR is delayed by some dependencies.